src/settings/components/Profile.tsx

```diff
@@ -660,9 +660,9 @@ const Profile: FunctionComponent<
 			}
 
 			case 'temp_access': {
-				const tempAccess = get(user, 'temp_access.current.status');
+				const tempAccess = get(profile, 'user.temp_access');
 
-				return tempAccess === 1
```

Comment:
this shows the correct value in the table, but didn't you also say something about, deleting the temp access entry in the database when someone gets permenant access?

Where is the code that does that deleting?